broker_test.go

```diff
@@ -19,8 +23,10 @@ type mockOrganizationsClient struct {
 }
 
 func (m mockOrganizationsClient) CreateAccount(input *organizations.CreateAccountInput) (*organizations.CreateAccountOutput, error) {
+	var id = "car-999999999999"
```

Comment:
Nitpick: common practice in Go is to use ["short assignment"](https://tour.golang.org/basics/10) for variables like this. In other words, this line would change to

```go
id := "car-999999999999"
```

Ditto for the other `var` usage in this file.